.gitignore

```diff
@@ -0,0 +1 @@
+.idea
```

Comment:
I use the following gitignore:

```
**__pycache__**
.idea
venv*
```

Although, I haven't committed it up to now, I think it's a good idea. Please, add lines I mentioned above.